### DISCUSSION TITLE
LoadError: The required version 0.23.2 of `spinedb_api` could not be found in the current Python environment

### DISCUSSION THREAD

**Discussion by pietrobm** [spine-tools/SpineOpt.jl#747] [Support]
Hello everyone, 

I am running my spine model on a cluster, and yesterday evening it worked with no issues, but this morning I got the attached error. I made no changes neither to the cluster directory nor to the input database. I have checked online and, as far as I have seen, there were no recent updates in the Spine tools. I updated my local version and tried to run it again on the cluster, obtaining the same error. 

The sbatch I run is attached as well as an example of the Julia script.

I know this could be unrelated to Spine, but I figured I'd ask anyways. Any feedback or advice would be very helpful. I can provide all the additional information that is needed. Thank you.

[slurm-27978793.out.zip](https://github.com/spine-tools/SpineOpt.jl/files/12611044/slurm-27978793.out.zip)
[scripts.zip](https://github.com/spine-tools/SpineOpt.jl/files/12611123/scripts.zip)

**Comment by jkiviluo** [comment]
There were also some errors about Gurobi - did you try if it works with open solver?

@soininen Do you see anything obvious here?

**Reply by pietrobm** [reply]
Thanks for the reply. I tried to run it this morning (without any change whatsoever from yesterday) and it worked. Given this result, I am convinced the problem was not related to Spine, but rather to the HPC. I have checked if there were any updates/changes on the HPC recently, but there were none. 
I am not sure on what could've caused this, but since now everything seems to be running fine I guess the problem is fixed.

**Reply by soininen** [reply]
This could have been caused by Git tags I added yesterday that accidentally caused this type of errors during installation. The tags have been since replaced with "safe" ones.
